include/boost/math/special_functions/beta.hpp

```diff
@@ -911,13 +916,17 @@ template <>
 struct Pn_size<float>
 {
    static constexpr unsigned value = 15; // ~8-15 digit accuracy
+#ifndef BOOST_MATH_HAS_GPU_SUPPORT
```

Comment:
Do we still have no static_assert?  That's a shame :(

Reply:
The `static_assert` should be fine, but the issue here is `operator[]` on std::array can't be used.

Reply:
Doh!